### PR TITLE
Add Electron support

### PR DESCRIPTION
**Pull request by IceSentry**
#37

### CODE REVIEW

File: src/components/Router.tsx [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
-      <Route path="/map" component={MapPage} />
-      <Route path="/config" component={ConfigPage} />
-    </Switch>
+    <>
+      <Redirect exact from="/" to="/teleop" />
+
+      <Switch>
+        <Route path="/teleop" component={Teleop} />
+        <Route path="/arm" component={Arm} />
+        <Route path="/map" component={MapPage} />
```
Comment (Ludoenso) [inline src/components/Router.tsx]: Map page? Did we define the different tabs for the UI? If not it might be best to take the time and define them.

**Comment by Ludoenso** [comment]
Also, correct me if I'm wrong, but now we are only targeting Windows as the platform for the WebUI since the space mouse doesn't have a functional driver for Linux?

File: src/components/Router.tsx (hunk 1 above)
Comment (IceSentry) [reply src/components/Router.tsx]: The tabs are defined in src/components/Header.tsx this was just a temporary router. I forgot to remove that route

**Comment by IceSentry** [comment]
> Also, correct me if I'm wrong, but now we are only targeting Windows as the platform for the WebUI since the space mouse doesn't have a functional driver for Linux?

Good question, it doesn't really matter right now, for everything else, but the spacemouse is indeed only supported in windows currently. I'll still try to keep the ui OS agnostic as much as possible.

**Comment by Ludoenso** [comment]
> 
> 
> > Also, correct me if I'm wrong, but now we are only targeting Windows as the platform for the WebUI since the space mouse doesn't have a functional driver for Linux?
> 
> Good question, it doesn't really matter right now, for everything else, but the spacemouse is indeed only supported in windows currently. I'll still try to keep the ui OS agnostic as much as possible.

File: README.md [hunk 2]
```diff
@@ -121,26 +122,23 @@ In the project directory, you can run:
...
 #### `npm run build`
 
 Builds the app for production to the `build` folder.
 It correctly bundles React in production mode and optimizes the build for the best performance.
 
 The build is minified and the filenames include the hashes.
-Your app is ready to be deployed!
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+Also builds an electron app installer for linux in windows in the folder `/dist`.
```
Comment (Ludoenso) [inline README.md]: Humm `for linux in windows in` Do you mean for WSL?

Comment (Ludoenso) [reply README.md]: maybe `and` windows?

Comment (IceSentry) [reply README.md]: oops, meant to write `linux and windows`

**Comment by LazyEngineerToBe** [comment]
> Also, correct me if I'm wrong, but now we are only targeting Windows as the platform for the WebUI since the space mouse doesn't have a functional driver for Linux?

If we continue on the electron path, we'll be able to use npm packages and there's one for the space mouse. Therefore, we might be able to achieve an os-agnostic UI. But that's not a priority for the moment.

**Pull request merged**